inlong-manager/manager-dao/src/main/resources/mappers/InlongStreamEntityMapper.xml

```diff
@@ -258,18 +258,18 @@
         stream.data_separator, stream.data_escape_char, stream.sync_send, stream.daily_records,
         stream.daily_storage, stream.peak_records, stream.max_length, stream.storage_period,
         stream.status, stream.creator, stream.modifier, stream.create_time, stream.modify_time
-        from inlong_stream stream, inlong_group group, workflow_approver approver
+        from inlong_stream stream, inlong_group grp, workflow_approver approver
```

Comment:
not necessary change

Reply:
It is the keyword in the h2 database:

<img width="1457" alt="image" src="https://user-images.githubusercontent.com/31994335/169439232-2d2b5418-ead4-453f-9dab-7b3739da584e.png">